### ISSUE TITLE
Wandering traders cannot be disabled in Killaura

### ISSUE THREAD

**Issue by hrmorley34** [Wurst-Imperium/Wurst7#175]
## Description
When using killaura, killauralegit or multiaura (and probably others too), there is no way to filter out wandering traders, as they are not considered animals or villagers (or for that matter any of the categories).

## Steps To Reproduce
1. Enable killaura
2. Turn on Filter animals and Filter villagers
3. Stand near a wandering trader
Note: while I was using many mods, I did also test on a fresh install/world with only Fabric Loader 0.9.1+build.205, Fabric API 0.17.0+build.386 and Wurst Client v7.4-MC1.16.1 (launched through MultiMC).

## Screenshots / Videos (Optional)
![image](https://user-images.githubusercontent.com/10192165/90127778-a3eaea00-dd5d-11ea-83a8-3f3dcf8b47c3.png)

## Crash Report
(manually triggered through F3-C)
```
---- Minecraft Crash Report ----
// You should try our sister game, Minceraft!

Time: 26/07/20 22:38
Description: Unexpected error

java.lang.NullPointerException: Unexpected error
	at net.minecraft.class_757.method_3188(class_757.java:692)
	at net.minecraft.class_757.method_3192(class_757.java:540)
	at net.minecraft.class_310.method_1523(class_310.java:1021)
	at net.minecraft.class_310.method_1514(class_310.java:654)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_757.method_3188(class_757.java:692)

-- Affected level --
Details:
	All players: 0 total; []
	Chunk stats: Client Chunk Cache: 729, 0
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 0 game time, 0 day time
	Server brand: ~~ERROR~~ NullPointerException: null
	Server type: Non-integrated multiplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:449)
	at net.minecraft.class_310.method_1587(class_310.java:2181)
	at net.minecraft.class_310.method_1514(class_310.java:677)
	at net.minecraft.client.main.Main.main(Main.java:215)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- System Details --
Details:
	Minecraft Version: 1.16.1
	Minecraft Version ID: 1.16.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_261, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 770626832 bytes (734 MB) / 1744830464 bytes (1664 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 8
	JVM Flags: 9 total; -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx2048m
	Fabric Mods: 
		autoconfig1u: Auto Config v1 Updated 3.2.0-unstable
		carpet: Carpet Mod in Fabric 1.4.0
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-config2: Cloth Config v4 4.5.6
		connected_block_textures: Connected Block Textures 0.1.0+1.16
		craftpresence: CraftPresence 1.6.6
		fabric: Fabric API 0.15.1+build.380-1.16.1
		fabric-api-base: Fabric API Base 0.1.3+12a8474c02
		fabric-biomes-v1: Fabric Biomes (v1) 0.2.7+059ea86602
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+c6a8ea8902
		fabric-command-api-v1: Fabric Command API (v1) 1.0.8+5ce5339802
		fabric-commands-v0: Fabric Commands (v0) 0.2.0+52d3083602
		fabric-containers-v0: Fabric Containers (v0) 0.1.8+045df74f02
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.9+059ea86602
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d02
		fabric-dimensions-v1: fabric-dimensions-v1 1.0.0+a71b305302
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.3.3+7066030f02
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+16acbe5b02
		fabric-item-api-v1: Fabric Item API (v1) 1.0.0+16acbe5b02
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.0+438f963602
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+f404f3be02
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+3fa9f7c502
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.1.0+013e49d202
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.0+386eb69e02
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+b764ce9902
		fabric-models-v0: Fabric Models (v0) 0.1.0+dfdb52d602
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.5+b50ffc7b02
		fabric-networking-v0: Fabric Networking (v0) 0.1.10+e00ecb5f02
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.5.6+a2d21ddd02
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.6.1+a2d21ddd02
		fabric-particles-v1: fabric-particles-v1 0.2.1+0a6f2a7002
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.3.8+7dba2d6c02
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.2.13+eae12eb802
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.3.3+0a55f37902
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.0.1+5a0f9a6002
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.3+b7f9825d02
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.8+b7084faa02
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.0+5341049002
		fabric-rendering-v1: Fabric Rendering (v1) 1.1.2+346247d702
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.2.6+f41e209802
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+97324d1102
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 0.2.7+a4c57d8e02
		fabric-textures-v0: Fabric Textures (v0) 1.0.4+eae12eb802
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.1.4+5794386e02
		fabricloader: Fabric Loader 0.9.0+build.204
		fabritone: Fabritone 1.5.3
		lambdynlights: LambDynamicLights 1.2.2+1.16.1
		lithium: Lithium 0.5.1
		malilib: MaLiLib 0.10.0-dev.21+arne.1
		minecraft: Minecraft 1.16.1
		modmenu: Mod Menu 1.14.5+build.30
		okzoomer: Ok Zoomer 3.0.1-20w12a
		phosphor: Phosphor 0.6.0+build.7
		replaymod: Replay Mod 1.16.1-2.4.0
		shulkerboxtooltip: Shulker Box Tootip 2.1.0+1.16.1
		spruceui: SpruceUI 1.5.8
		wurst: Wurst Client v7.4-MC1.16.1
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Intel(R) UHD Graphics 620 GL version 4.5.0 - Build 25.20.100.6577, Intel
	GL Caps: Using framebuffer using OpenGL 3.0
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Resource Packs: 
	Current Language: English (United Kingdom)
	CPU: 8x Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz
```

**Comment by Neilpotent** [comment]
This bug appears to occur with armor stands as well. They seemed to be missed out in multi-aura's setting menu

**Comment by hrmorley34** [comment]
Fixed in v7.5/1.16.3
